controllers/openstackmachine_controller.go

```diff
@@ -430,7 +459,35 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, scope
 	return ctrl.Result{}, nil
 }
 
-func (r *OpenStackMachineReconciler) getOrCreate(logger logr.Logger, cluster *clusterv1.Cluster, openStackCluster *infrav1.OpenStackCluster, machine *clusterv1.Machine, openStackMachine *infrav1.OpenStackMachine, computeService *compute.Service, userData string) (*compute.InstanceStatus, error) {
+func (r *OpenStackMachineReconciler) getOrCreatePorts(logger logr.Logger, clusterName string, openStackCluster *infrav1.OpenStackCluster, securityGroups []infrav1.SecurityGroupFilter, openStackMachine *infrav1.OpenStackMachine, networkingService *networking.Service) ([]infrav1.PortStatus, error) {
+	// TODO(emilien) implement portsStatus where we check if the machine has ports created and part of the OpenStackMachine.Status
+	// Check `GetInstanceStatusByName` as it's done for instances. For each port found in Status, we might want get the port name and verify there is no duplicate.
+	// If Status is empty, we create the ports and add them to the Status.
+	// If Status is not empty, we try to adopt the existing ports by checking if the ports are still there and if not, we create them and add them to the Status.
+
+	// For now, we create the ports.
```

Comment:
this line seems not consistent to above 2 lines comments?

Reply:
I'll remove it. Thanks